db/schema.rb

```diff
@@ -310,6 +310,7 @@
     t.string "slug"
     t.string "status", default: "active"
     t.datetime "updated_at", null: false
+    t.index ["slug"], name: "index_chat_channels_on_slug", unique: true
```

Comment:
this should be removed, my suggestion is to open `psql PracticalDeveloper_development` and `DROP INDEX index_chat_channels_on_slug` as it's a remnant of some old change we don't have in production anymore